README.md

```diff
@@ -62,6 +62,16 @@ a provided `session_name`.
 
 `bind C-r "run-shell 'tms refresh'"`.
 
+### The `tms kill` command
+
+Using this command you can kill current tmux session and automatically jump to another. The config
+could look like this `bind C-k command-prompt -p "Kill current session? (y/N): " "run-shell 'test
+\"%1\" = y && tms kill || true'"`. Then when using leader+C-k you have to confirm killing of the
```

Comment:
I think I'm using this with just confirm, then you don't need to check for y afterwards
```suggestion
could look like this `bind C-k confirm -p "Kill current session? (y/N): " "run-shell 'tms kill || true'"`. Then when using leader+C-k you have to confirm killing of the
```